circuit_maintenance_parser/parser.py

```diff
@@ -161,3 +164,32 @@ def clean_line(line):
             line = line.strip()
         # TODO: below may not be needed if we use `quopri.decodestring()` on the initial email file?
         return line.replace("=C2", "").replace("=A0", "").replace("\r", "").replace("=", "").replace("\n", "")
+
+
+class Subject(Parser):
+    """Parse data from subject or email."""
+
+    _data_types = ["email-header-subject"]
+
+    def parse(self, subject: bytes) -> List[Dict]:
+        """Execute parsing."""
+        result = []
+
+        data_base: Dict[str, Union[int, str, Iterable]] = {}
+        try:
+            for data in self.parse_subject(self.bytes_to_string(subject), data_base):
```

Comment:
I would simplify this parser. It's not necessary to reproduce the same schema like the other, where the `data_base` it's incremented, in this case you just get an attribute, so no need to iterate.
Example: https://github.com/networktocode/circuit-maintenance-parser/pull/59/files#diff-b5df3420c13a1de156bbb25c6f41d0448704d43bd9feee7f0bcfe131804f0f8eR172